datamol/graph.py

```diff
@@ -179,8 +179,7 @@ def reorder_mol_from_template(
     mol_template: dm.Mol,
     enforce_atomic_num: bool = False,
     enforce_bond_type: bool = False,
-    allow_ambiguous_match: bool = False,
-    allow_ambiguous_hs_only: bool = False,
+    allow_ambiguous_match: str = "No",
```

Comment:
to avoid ambiguity,  use a variable name that will not be interpreted as a False/True (e.g `ambiguous_match_mode`, etc)